app/components/dashboard/DcChart.vue

```
@@ -954,7 +758,7 @@
 </template>
 
 <script setup>
-import {onMounted, ref, watch, reactive} from "vue";
+import {onMounted, ref, watch, reactive, nextTick} from "vue";
```

ESLint

[error] A space is required after '{'.

(@stylistic/object-curly-spacing)